rules/how-to-monetize-apps/rule.md

```diff
@@ -14,66 +16,87 @@ redirects:
+* Your app is a mobile or web app (desktop users are less tolerant of in-app ads)
+* Your app has a strong re-use strategy (e.g., 'addictive' games, or diary-like apps that require or support daily use)
+* Your target demographic is less likely to pay to use an app (and less bothered by ads)
 
+**NOTE:** If you include ads in your app, you should consider a freemium model, so that users can pay to remove them.
 
-::: bad
-Bad example: same prices for everyone  
+::: good
+![Figure: The Cronometer app is free, but shows ads to users.](cronometer.png)
+**Good Example:** Cronometer offers a paid and free tier. On the free tier, users see ads, but can pay to remove them.
 :::
 
+### Paid content
+Some apps provide the app for free, but content within the app is paid for. This is a good model for content creators, or an excellent model for content platforms with user generated content. This might be the right approach for you if:
+* Your content is your product, and your app is just a delivery vessel
+* You have a lot of user generated content (and your users want to monetize that content too)
+* You are regularly adding new content to your app
 
 ::: good
-Good example: capture data eg. What device and do data mining to set different prices 
-
+![Figure: The Zinio app is a digital magazine platform. The app is free, but users pay to subscribe to a magazine or to purchase individual issues.](zinio.png)
+**Good Example:** The Zinio app is a digital magazine platform. The app is free, but users pay to subscribe to a magazine or to purchase individual issues.
 :::
 
+### Loss leader
+This approach is uncommon, but occasionally an app developer might want to give an app away for free to entice users to try another, revenue generating app. Examples include showcase apps for UI control library vendors, like [Syncfusion](https://apps.apple.com/us/app/syncfusion-dashboard-mobile/id1157300621), [Telerik](https://apps.apple.com/us/app/telerik-ui-for-xamarin-samples/id1083924868) or [Grial Kit](https://apps.apple.com/us/app/grial-uikit/id1099501310).
 
+### Data mining
+Some apps offer functionality to users for free, and in exchange gather information about users for sale to advertisers or other interested parties. Tiktok, one of the most successful apps in the world, is a [surveillance app that mines data about users, their connections, and environment](https://www.afr.com/policy/foreign-affairs/tiktok-s-alarming-excessive-data-collection-revealed-20220714-p5b1mz). Some apps are more extreme than others, and the breadth and depth of dat acquisition can vary.
```

Comment:
Add some venomous bullet-points about this. It's by far the worst offender of bad monetization practices.

Reply:
Done.